### DISCUSSION TITLE
Appeal

### DISCUSSION THREAD

**Discussion by RichtofenFemboiOwO** [nzp-team/nzportable#677] [General]
Hey cypress i just wanted to know if i could appeal my ban i wasn't trying to be mean i was joking around although i completely understand why u ban me i promise not to pose memes and general and apologize for saying the comeback i want to be apart of the community and plan to make videos on this game 

Plz unban me im sorry : (

**Comment by Paindeluxe** [comment]
While I do think that a punishment was deserved a ban did go a bit too far. Why not just give him a timeout?

**Comment by MotoLegacy** [comment]
It's not my job to babysit. I made it clear several times in the Matrix space that you were actively harming the flow of conversation, sometimes attempting to attention seek in general ignoring the obvious support query, ("the word 'nazi' in your game is offensive i'm going to cancel you on twitter").

I don't work on this project to harbor low effort communication, nor was it the intention with the Discord+Matrix+IRC bridge. Sorry!

**Reply by RichtofenFemboiOwO** [reply]
oh come on thats clearly a joke breh

**Reply by MotoLegacy** [reply]
That's the point. Someone was clearly trying to engage in a serious conversation and after several times of telling you to go away you felt the need to continue to interrupt the discussion. Locking this thread.
